lms/wsgi.py

```diff
@@ -8,28 +8,32 @@
 ``WSGI_APPLICATION`` setting.
 """
 
+import os  # lint-amnesty, pylint: disable=wrong-import-order, wrong-import-position
+
+# Disable PyContract contract checking when running as a webserver
+import contracts  # lint-amnesty, pylint: disable=wrong-import-order, wrong-import-position
+# This application object is used by the development server
+# as well as any WSGI server configured to use this file.
+from django.core.wsgi import \
+    get_wsgi_application  # lint-amnesty, pylint: disable=wrong-import-order, wrong-import-position
+
+import lms.startup as startup  # lint-amnesty, pylint: disable=wrong-import-position
 # Patch the xml libs
 from safe_lxml import defuse_xml_libs
+from xmodule.modulestore.django import modulestore  # lint-amnesty, pylint: disable=wrong-import-position
```

Comment:
FYI, we believe that these import reorderings were the cause of the stage deploys failing. I am not sure this file gets used in devstack, which is why local testing may not have caught it.